bedrock/settings/base.py

```diff
@@ -231,6 +231,7 @@ def data_path(*args):
 
 # Global L10n files.
 FLUENT_DEFAULT_FILES = [
+    "affiliate.ftl",
```

Comment:
Nit: remove filename suffix for consistency?
```suggestion
    "affiliate",
```